docs/superpowers/specs/assets/2026-06-18-mobile-ux/mockup.html

```
@@ -1,309 +1,895 @@
+        <div style="height: 10px"></div>
+        <div class="lrow current">
+          <span>Network Closet</span
+          ><span class="meta">2 racks &middot; current</span>
+        </div>
+        <div class="lrow">
+          <span>Home Lab</span><span class="meta">1 rack</span>
+        </div>
+        <div class="lrow">
+          <span>Studio Rack</span><span class="meta">3 racks</span>
+        </div>
+        <button class="newrow">+ New layout</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)